src/Hosting/TestHost/test/ResponseBodyTests.cs

```diff
@@ -46,6 +48,80 @@ public async Task BodyWriter_StartAsyncGetMemoryAdvance_AutoCompleted()
             Assert.Equal(length, bytes.Length);
         }
 
+        [Fact]
+        public async Task BodyStream_fails_synchronous_write_when_not_enabled()
```

Comment:
Please follow the naming convention here. `BodyStream_SyncDisabled_WriteThrows`

Reply:
Thank you. Wasn't aware that there was naming convention. I looked around at other tests and found there are multiple naming conventions.